modules/admin_manual/pages/configuration/server/config_apps_sample_php_parameters.adoc

```diff
@@ -824,29 +811,37 @@ it's assumed that all users belong to the same domain, otherwise
 "user001@my.dom.com" will be mapped to the same windows user as
 "user001@not.mine.eu".
 
-- `CustomFile`: This mapping is reserved and should only be used on explicit request of ownCloud support.
+- `EALdapAttr`: Use ownCloud's user extended attributes to map the ownCloud
+user to the target LDAP attribute. The mapping has an "attr" parameter in order
+to select the LDAP attribute you want to use. Note that it's required that
+the user_ldap app exposes the chosen attribute which requires user_ldap 0.19.0+.
+In case ownCloud can't map the user, an error will be thrown.
```

Comment:
```suggestion
If ownCloud can't map the user, an error will be thrown.
```

Reply:
Same as below,will do in core and transport it back

Reply:
This is not a really important change, so don't worry too much about having to push another cocre PR.

Reply:
already in pushing state... 😄

Reply:
pls approve/merge: https://github.com/owncloud/core/pull/41134